docs/docs/deploying-to-digitalocean-droplet.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Deploying to a DigitalOcean Droplet
+---
+
+This guide walks through how to deploy and host your Gatsby site on a [DigitalOcean Droplet](https://www.digitalocean.com/products/droplets/).
```

Comment:
```suggestion
This guide walks through how to deploy and host your Gatsby site on a [DigitalOcean Droplet](https://www.digitalocean.com/products/droplets/) with Ubuntu and Nginx.
```

Recommend to make the technology for this more explicit at the beginning, since there are other options it doesn't cover.